config-linux.md

```diff
@@ -196,6 +196,39 @@ For example, to run a new process in an existing container without updating limi
 
 Runtimes MAY attach the container process to additional cgroup controllers beyond those necessary to fulfill the `resources` settings.
 
+### Cgroup ownership
+
+Runtimes MAY change (or cause to be changed) the owner of the
+container's cgroup to the host uid that maps to uid 0 in the
+container's user namespace, according to the following rules.
+
+Runtimes MUST NOT change the ownership of container cgroups when
+cgroups v1 is in use.  Cgroup delegation is not secure in cgroups
+v1.
+
+Runtimes SHOULD change the cgroup ownership if and only if the
+cgroup filesystem is to mounted read/write; that is, when the
+configuration's `mounts` array contains an object where:
+
+- The `source` field is equal to `"cgroup"`
+- The `destination` field is equal to `"/sys/fs/cgroup"`
+- The `options` field does not contain the value `"ro"`
+
+If the configuration does not specify such a mount, the runtime MUST
+NOT change the cgroup ownership.
+
+A runtime that changes the cgroup ownership MUST only change the
+ownership of the container's cgroup directory and the following
+files within that directory:
+
+- `cgroup.procs`
+- `cgroup.subtree_control`
+- `cgroup.threads`
+
+The runtime MUST NOT change the ownership of any other files.
+Changing other files may allow the container to elevate its own
+resource limits or perform other unwanted behaviour.
```

Comment:
Same comment about MUST and MUST NOT here. It should be SHOULD and SHOULD NOT. It's also entirely possible that it would be safe to allow the container to control the cgroup it's in (for instance if the runtime put the container in a nested cgroup and enforced the restrictions in the parent -- AFAIK this kind of thing is necessary to run systemd in containers so making it something you are not allowed to do is way too extreme IMHO).

Reply:
relaxed to SHOULD.